README.md

```diff
@@ -1,9 +1,190 @@
+
+```bash
+rosrun clever_tools interactive.py
+```
+
+### create_route.py
+
+Create flying route by moving drone in space and recording its coordinates by triggering RC pitch stick (channel 2).
+
+```bash
+usage: rosrun clever_tools create_route.py [-h] [-f FRAME_ID] [filename]
+
+positional arguments:
+  filename              Filename of route csv table. Default is route.csv.
+
+optional arguments:
+  -h, --help            show help message and exit
+  -f FRAME_ID, --frame_id FRAME_ID
+                        Coordinates will be recorded relative to frame_id
+                        parameter. Default is map.
+```
+
+### fly_route.py
+
+Fly route from the csv table with x y z values.
+
+```bash
+usage: rosrun clever_tools fly_route.py [-h] [-r] [-z HEIGHT] [-f FRAME_ID] [-s SPEED] [filename]
+
+positional arguments:
+  filename              Filename of the route csv table
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -r, --repeat          Program will repeat the route infinite if you use this
+                        arg.
+  -z HEIGHT, --height HEIGHT
+                        Height of flight. Default is 1 m. If height is 'nan'
+                        then the drone will takeoff to 1 m and fly on the
+                        route point heights.
+  -f FRAME_ID, --frame_id FRAME_ID
+                        Coordinates will be used relative to frame_id
+                        parameter. Default is map.
+  -s SPEED, --speed SPEED
+                        Copter speed. Default is 1 m/s.
+```
+
+### test_flight.py
+
+Just example node of flight: takeoff to 1m, fly forward 1m, and land.
+
+## Python modules
+
+### flight_routines
+
+#### Usage example
+
+```
+import rospy
+from flight_routines import *
+
+rospy.init_node('flight routines node')
+
+# copter parameters
+
+speed = 1
+z = 1
+frame = 'map'
+x0 = get_telemetry(frame_id = frame).x
+y0 = get_telemetry(frame_id = frame).y
+
+# flight program
+
+takeoff(z)                                                  # takeoff
+reach_point(x=x0, y=y0+1, z=z, speed=speed, frame_id=frame) # flight 1m toward
+land()
+```
+
+
+#### Globals
+
+```bash
+TOLERANCE = 0.2             # m
+SPEED = 1.0                 # m/s
+TAKEOFF_SPEED = 1.0         # m/s
+TAKEOFF_HEIGHT = 1.0        # m
+LOCAL_FRAME_ID = 'map'
+```
+
+#### Service proxies
+
+Most services described [here](https://clever.copterexpress.com/ru/simple_offboard.html)
+
+* `navigate` to service `/navigate`
+* `navigate_global` to service `/navigate_global`
+* `set_position` to service `/set_position`
+* `set_velocity` to service `/set_velocity`
+* `set_attitude` to service `/set_attitude`
+* `set_rates` to service `/set_rates`
+* `get_telemetry` to service `get_telemetry`
+* `land` to service `/land`
+* `arming` to service `/mavros/cmd/arming`
+* `set_mode` to service `/mavros/set_mode`
+
+#### Functions
+
+```bash
+get_distance(x1, y1, z1, x2, y2, z2)
```

Comment:
Shouldn't it be a list of something? Otherway this is just code samples within the text.

Reply:
I don't understand this